server/game/cards/03_TWI/events/InPursuit.ts

```diff
@@ -0,0 +1,44 @@
+import { EventCard } from '../../../core/card/EventCard';
+import AbilityHelper from '../../../AbilityHelper';
+import { RelativePlayer, WildcardCardType, WildcardZoneName } from '../../../core/Constants';
+
+export default class InPursuit extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6401761275',
+            internalName: 'in-pursuit'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
```

Comment:
I have a feeling we need to rework this one a bit to use an ifYouDo

Reply:
ahhh, yeah, that is a good call. Let me look into that method.

Reply:
Yeah exactly correct. As Chaqa notes below, this version won't work correctly if the friendly unit is already exhausted.

Reply:
I fixed the card to have an `IfYouDo` and ensured you could only exhaust a readied-friendly unit. I have still allowed the exhausting of any enemy unit, regardless of state, as it seems to match the rule, even though if you pick an exhausted enemy unit, the card does nothing.